src/index.js

```diff
@@ -2,16 +2,16 @@ import template from 'babel-template';
 
 export default function ({ types: t }) {
 
-  const requireIdentifier = t.identifier('$__require');
+  const requireIdentifier = t.identifier('require');
 
   const buildTemplate = template(`
     SYSTEM_GLOBAL.registerDynamic(MODULE_NAME, [DEPS], true, BODY);
   `);
 
   const buildFactory = template(`
-    (function ($__require, exports, module) {
+    (function (require, exports, module) {
       BODY
-      return module.exports;
+      MODULE_EXPORT
```

Comment:
I don't think so actually. Let's try it through the tests, and see how that goes, and if so we can always revert?